DqtNotifications/terraform/app.tf

```diff
@@ -0,0 +1,153 @@
+locals {
+  auth_rule_name = "SendAndListenSharedAccessKey"
+  sku            = var.enable_private_endpoint == true ? "Premium" : var.sku
+  capacity       = local.sku == "Premium" && var.capacity <= 0 ? 1 : var.capacity
+       }
+
+# App Sercie Plan
+resource "azurerm_app_service_plan" "app_service_plan" {
+  name                = local.app_service_plan_name
+  location            = data.azurerm_resource_group.rgsb.location
+  resource_group_name = data.azurerm_resource_group.rgsb.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Dynamic"
+    size = "Y1"
+  }
+  
```

Comment:
1 line of whitespace is sufficient, there are other places where some whitespace needs removing